README.md

```diff
@@ -166,543 +119,10 @@ There are two parameters:
 * a space number (it could just as easily have been a space name), and
 * a tuple.
 
-## Help
-
```

Comment:
If you want to contact the developers of `go-tarantool` with problems, 
create an issue at [tarantool/go-tarantool](http://github.com/tarantool/go-tarantool/issues).

The developers of the [Tarantool server](http://github.com/tarantool/tarantool)
will also be happy to provide you advice or receive your feedback.

Reply:
It is  not relevant to proposed changes.